db/insertData.js

```diff
@@ -0,0 +1,58 @@
+const db = require('./index');
+//const arango = require('./index').arango;
+const axios = require('axios');
+const fs = require('fs');
+const csv = require('csv-parser');
+const path = require('path');
+const neatCsv = require('neat-csv');
+
+const COLLECTIONS = ['games', 'edges', 'traits'];
+
+const collections = {
+  games: db.collection('games'),
+  traits: db.collection('traits'),
+  edges: db.collection('edges')
+};
+
+const loadData = async () => {
+  let files = [];
+  let dir = fs.readdir(path.resolve(__dirname, 'data'), (err, inFiles) => (files = inFiles));
+  let start = Date.now();
+
+  await setupCollections();
+  console.log('Collections setup.');
+
+  // let filteredFiles = files.filter((file) => file.match(COLLECTIONS[i]));
+  for (let j = 0; j < files.length; j++) {
+    let data = fs.readFileSync(path.resolve(__dirname, 'data', files[j]), 'utf8');
+    let collection = files[j].match(/(.+?)(?=\d|\.)/)[0];
+    data = await neatCsv(data);
+    await insertData(data, collections[collection]);
```

Comment:
What happens if data insertions into the DB fail halfway through, or your power goes out? Do you end up with a half-done collection of documents? Maybe Arango's transaction functionality would be a good failsafe, if you haven't looked into it already.